app/views/events/index.html.erb

```diff
@@ -0,0 +1,72 @@
+        <p class="display-6 text-danger font-weight-bold">Keep in touch with</p>
+        <p class="display-3 font-weight-bold">virtual events</p>
+      </div>
+      <div class="col-md-6">
+        <img class="img-fluid" src="https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F99891082%2F174963144412%2F1%2Foriginal.20200430-164830?w=1016&auto=format%2Ccompress&q=75&sharp=10&s=be5cc9aa4646e8daeb55c374bc923a05" alt="">
+      </div>
+
+
+      <!--end main content-->
+
+      <div id="overlay"></div>
+    </div>
+  </div>
+  <div class="container mb-5">
+    <div class="col-12">
+      <nav>
+        <div class="nav nav-tabs nav-fill" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-home-tab" data-toggle="tab" href="#nav-home" role="tab" aria-controls="nav-home" aria-selected="true">Past Events</a>
+            <a class="nav-item nav-link" id="nav-profile-tab" data-toggle="tab" href="#nav-profile" role="tab" aria-controls="nav-profile" aria-selected="false">Upcoming events</a>
+        </div>
+      </nav>
+      <div class="tab-content" id="nav-tabContent">
+          <div class="tab-pane fade show active" id="nav-home" role="tabpanel" aria-labelledby="nav-home-tab">
+            <% @past.each do |event| %>
+              <div class="row mt-4 event p-4 pointer">
+                <div class="col-md-4">
+                  <img class="img-fluid" src="https://process.filestackapi.com/AtM7HNKzQZ6u2HxwJF1Jiz/compress/quality=value:90/0tTy4z3lTbCkw18ehjQ8" alt="">
+                </div>
+                <div class="col-md-8">
+                  <div class="date">
+                    <span class="text-uppercase text-danger"><%= event.date %></span>
+                  </div>
+                  <div class="event-title">
+                    <h3 class="font-weight-bold">
+                      <%= link_to event.title, event_path(event), class: "text-dark" %>
+                    </h3>
+                  </div>
+                </div>
+              </div>
+            <% end %>
+
+          </div>
+          <div class="tab-pane fade" id="nav-profile" role="tabpanel" aria-labelledby="nav-profile-tab">
+            <% @upcoming.each do |event| %>
+              <div class="row mt-4 event p-4 pointer">
+                <div class="col-md-4">
+                  <img class="img-fluid" src="https://process.filestackapi.com/AtM7HNKzQZ6u2HxwJF1Jiz/compress/quality=value:90/0tTy4z3lTbCkw18ehjQ8" alt="">
+                </div>
+                <div class="col-md-8">
+                  <div class="date">
+                    <span class="text-uppercase text-danger"><%= event.date %></span>
+                  </div>
+                  <div class="event-title">
+                    <h3 class="font-weight-bold">
+                      <%= link_to event.title, event_path(event), class: "text-dark" %>
+                    </h3>
+                  </div>
+                </div>
+              </div>
+            <% end %>
```

Comment:
- [ ] You should be displaying upcoming events by using event model scope in the show action of your User's controller. 
<img width="807" alt="Screen Shot 2020-12-10 at 8 28 56 PM" src="https://user-images.githubusercontent.com/48804258/101849747-55d8ce00-3b26-11eb-8aae-f990d1e64272.png">